assembl/locale/assembl-v2/es.po

```diff
@@ -11,13 +11,14 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: Poedit 2.0.7\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+:MEMOQLABEL_WARN_EMPTYMSGID_1
```

Comment:
Ugh, would be nice to remove these, they are vendor specific keys that have little purpose.